openapi/resource-server.yaml

```diff
@@ -641,7 +628,6 @@ paths:
                     - receiver
                     - sendAmount
                   additionalProperties: false
-
```

Comment:
I think the quote still has `sendAmount`, are we keeping that?

Reply:
No, it's all `debitAmount` now, since the quote also has fees included.